package.json

```diff
@@ -16,105 +16,106 @@
     "start:development:client": "node client/scripts/start.js"
   },
   "dependencies": {
-    "argon2": "^0.16.2",
-    "autoprefixer": "^7.1.2",
-    "axios": "^0.16.2",
-    "babel-core": "6.25.0",
+    "argon2": "^0.17.3",
+    "autoprefixer": "^8.2.0",
+    "axios": "^0.18.0",
+    "babel-core": "6.26.0",
     "babel-eslint": "7.2.3",
-    "babel-loader": "^7.1.1",
-    "babel-plugin-transform-react-jsx": "^6.8.0",
-    "babel-plugin-transform-runtime": "^6.15.0",
-    "babel-polyfill": "^6.9.1",
-    "babel-preset-es2015": "^6.18.0",
-    "babel-preset-react-app": "^3.0.1",
-    "babel-preset-stage-2": "^6.18.0",
-    "babel-runtime": "^6.25.0",
+    "babel-loader": "^7.1.4",
+    "babel-plugin-transform-react-jsx": "^6.24.1",
+    "babel-plugin-transform-runtime": "^6.23.0",
+    "babel-polyfill": "^6.26.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react-app": "^3.1.1",
+    "babel-preset-stage-2": "^6.24.1",
+    "babel-runtime": "^6.26.0",
     "bcrypt": "^1.0.3",
```

Comment:
I tried to build with the upstream git + this PR + "bcrypt": "^2.0.1", with nodejs 10.0.0 in order to address https://github.com/jfurrow/flood/issues/656

Reply:
This should be a benign change https://github.com/kelektiv/node.bcrypt.js#migrating-from-v10x. I worked for me.